blog/services/image_upload_service.py

```
@@ -0,0 +1,19 @@
+    "image/jpeg",
+    "image/png",
+    "image/gif",
+    "image/webp",
+}
+
+MAX_IMAGE_SIZE = 5 * 1024 * 1024  # 5MB
+
+
+def validate_uploaded_image(uploaded_file):
+    if uploaded_file.content_type not in ALLOWED_IMAGE_CONTENT_TYPES:
+        raise ValidationError("이미지 파일만 업로드할 수 있습니다.")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)